scripts/linux/runE2ETest.sh

```diff
@@ -84,7 +85,7 @@ function prepare_test_from_artifacts() {
     mkdir -p "$working_folder"
 
     declare -a pkg_list=( $iotedged_artifact_folder/*.deb )
-    iotedge_package="${pkg_list[@]:0}"
+    iotedge_package="${pkg_list[*]:0}"
```

Comment:
I assume you did this because of this warning:

`SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.`

But the point is not to use `*`, because it doesn't make sense to `:0` in either case.

The original intent was to assign the first element of the array, so it should be `${pkg_list[0]}`

Reply:
":0" means starting at index 0.  And installation needs to provide both iotedge and libiohsm packages.  I updated to remove ":0" to avoid confusing.

Reply:
You're right. Even though it's called `archivePath`, we pass the value to `dpkg -i` without any quoting, so it does work even if it has multiple space-separated paths.